### DISCUSSION TITLE
my characters are facing the opposite way.

### DISCUSSION THREAD

**Discussion by vinnieyeayea** [CodenameCrew/CodenameEngine#332] [Q&A] [answered]
how do i fix that? (also the player is floating, i also fixed the opponent) they're also not playing their note animations.

**Comment by NexIsDumb** [comment, marked as answer]
go inside the character editor, you’ll find everything there (in the main menu press 7)
the sing animations must be named like singRIGHT or singLEFT or for the misses for example singUPmiss
